Kickstarter-iOS/Views/Controllers/LiveStreamCountdownViewController.swift

```diff
@@ -0,0 +1,348 @@
+import KsApi
+import Library
+import LiveStream
+import Prelude
+import ReactiveSwift
+import UIKit
+
+public final class LiveStreamCountdownViewController: UIViewController {
+  @IBOutlet private weak var activityIndicatorView: UIActivityIndicatorView!
+  @IBOutlet private weak var creatorAvatarBottomConstraint: NSLayoutConstraint!
+  @IBOutlet private weak var creatorAvatarImageView: UIImageView!
+  @IBOutlet private weak var creatorAvatarWidthConstraint: NSLayoutConstraint!
+  @IBOutlet private var countdownColons: [UILabel]!
+  @IBOutlet private weak var countdownStackView: UIStackView!
+  @IBOutlet private weak var daysSubtitleLabel: UILabel!
+  @IBOutlet private weak var daysTitleLabel: UILabel!
+  @IBOutlet private weak var detailsStackViewBackgroundView: UIView!
+  @IBOutlet private weak var detailsStackView: UIStackView!
+  @IBOutlet private weak var gradientView: GradientView!
+  @IBOutlet private weak var hoursSubtitleLabel: UILabel!
+  @IBOutlet private weak var hoursTitleLabel: UILabel!
+  @IBOutlet private weak var introLabel: SimpleHTMLLabel!
+  @IBOutlet private weak var liveStreamTitleLabel: UILabel!
+  @IBOutlet private weak var liveStreamParagraphLabel: UILabel!
+  @IBOutlet private weak var minutesSubtitleLabel: UILabel!
+  @IBOutlet private weak var minutesTitleLabel: UILabel!
+  @IBOutlet private weak var projectImageView: UIImageView!
+  @IBOutlet private weak var secondsSubtitleLabel: UILabel!
+  @IBOutlet private weak var secondsTitleLabel: UILabel!
+  @IBOutlet private weak var subscribeActivityIndicatorView: UIActivityIndicatorView!
+  @IBOutlet private weak var subscribeButton: UIButton!
+
+  private let eventDetailsViewModel: LiveStreamEventDetailsViewModelType = LiveStreamEventDetailsViewModel()
```

Comment:
Both the `LiveStreamCountDownViewController` and `LiveStreamContainerViewController` share this `LiveStreamEventDetailsViewModel`. It's perhaps not the most ideal situation but there is a fair bit of code duplication between the two view controllers so I chose to do it this way.

One thing to bear in mind is that the `LiveStreamEventDetailsViewModel` fetches the `LiveStreamEvent` model and updates the other view models with this.